.github/workflows/tools-python.yaml

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: TNG Technology Consulting GmbH
```

Comment:
Is this a thing that we want to keep?

Reply:
No, I changed it to `SPDX Contributors` and also updated the license header in the python files.